sanic/router.py

```diff
@@ -65,17 +66,17 @@ def my_route_with_type(request, my_param: my_type):
     routes_always_check = None
     parameter_pattern = re.compile(r'<(.+?)>')
 
-    def __init__(self):
+    def __init__(self, converters=REGEX_TYPES):
```

Comment:
From point of usability, will it be better to define custom_converters?
By custom_converters I mean extending string, int, number, alpha, path converters.

```
def __init__(self, custom_converters =None):
    self._converters = dict(REGEX_TYPES)
    if  custom_converters:
        self._converters.update(custom_converters)
```